app/models/user.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      name: { type: DataTypes.STRING, allowNull: false },
+      lastName: { type: DataTypes.STRING, allowNull: false },
+      email: { type: DataTypes.STRING, allowNull: false, unique: true },
+      password: { type: DataTypes.STRING, allowNull: false }
+    },
+    {}
```

Comment:
Remove properties if empty, check [this](http://docs.sequelizejs.com/manual/tutorial/models-definition.html#configuration), more used are `underscored` and `freezeTableName`